asp-core/src/test/java/de/jcup/asp/core/CryptoAccessTest.java

```diff
@@ -88,5 +88,18 @@ public void encrypted_string_can_be_reversed_by_another_crypto_access_with_wrong
 
     }
     
+    @Test
```

Comment:
When I add this test without any other changes (from this PR) to my eclipse instance and start the test it is green.

When I execute `./gradlew test` i also got no failing test.

Maybe this is, because my JVM is always starting with `file.encoding=UTF-8` and you maybe with `file.encoding=ISO-8859-1` ?

So tests seems not failing for somebody using UTF-8.

Reply:
Yes you need to have a different encoding like on Windows